src/common/components/link/Link.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import classNames from "classnames";
+
+import styles from "./link.module.scss";
+
+type Props = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, "children"> & {
+  children?: JSX.Element | string;
+  iconLeft?: JSX.Element;
+  className?: string;
+};
+
+export default function Link({
+  children,
+  iconLeft,
+  className,
+  ...props
+}: Props) {
+  return (
+    <a className={classNames(styles.link, className)} {...props}>
+      {iconLeft && iconLeft}
+      {children && children}
+    </a>
+  );
+}
```

Comment:
Shouldn't we use the ones that are configured in the core and added in the context?

``` javascript
export type Config = {
  siteName: string;
  currentLanguageCode: LanguageCodeEnum;
  copy: {
    breadcrumbNavigationLabel: string;
    breadcrumbListLabel: string;
    menuToggleAriaLabel: string;
    skipToContentLabel: string;
    openInExternalDomainAriaLabel: string;
    openInNewTabAriaLabel: string;
    closeButtonLabelText: string;
  };
  components: {
    A: (props: React.AnchorHTMLAttributes<HTMLAnchorElement>) => JSX.Element;
    Img: (props: React.ImgHTMLAttributes<HTMLImageElement>) => JSX.Element;
    Link?: (
      props: React.AnchorHTMLAttributes<HTMLAnchorElement>
    ) => JSX.Element;
    Head?: (props: { children: React.ReactNode }) => JSX.Element;
  };
  apolloClient?: ApolloClient<NormalizedCacheObject>;
  utils: {
    getIsHrefExternal: (href: string) => boolean;
  };
};
```